files/zh-cn/learn/tools_and_testing/client-side_javascript_frameworks/vue_getting_started/index.html

```diff
@@ -63,7 +63,7 @@ <h2 id="安装Vue">安装 Vue</h2>
 </ol>
 
 <div class="blockIndicator note">
-<p><strong>注意：如果您还没有安装上述软件，有关安装 NPM 和 Node.js 的方法在这里</strong></p>
+<p><strong>注意：如果您还没有安装上述软件，有关安装 NPM 和 Node.js 的方法<a href="/zh-CN/docs/Learn/Tools_and_testing/Understanding_client-side_tools/Command_line#%E6%B7%BB%E5%8A%A0%E5%B7%A5%E5%85%B7">在这里</a></strong></p>
```

Comment:
I have allow myself to correct the notecard:

```suggestion
<p><strong>备注：</strong>如果您还没有安装上述软件，有关安装 NPM 和 Node.js 的方法<a href="/zh-CN/docs/Learn/Tools_and_testing/Understanding_client-side_tools/Command_line#添加工具">在这里</a>。</p>
```